### DISCUSSION TITLE
product roadmap focusing on pair and fulfill (2017-2026)

### DISCUSSION THREAD

**Discussion by hyunjimoon** [Data4DM/BayesSD#173] [perspectives; sense-making]
research vision: OFE: #159, OIA: #174, BAE: #147 
educational training: #175 
<img width="1232" alt="image" src="https://github.com/Data4DM/BayesSD/assets/30194633/c6637254-531c-4e9f-b072-10235b41b1d1">

**Comment by hyunjimoon** [comment]
Q.
I'm curious how supply side uncertainty can be modeled and whether it is reasonable to think this becomes more important as we move toward monitor to control to plan (longer term, from Baichuan’s talk). 

I’ve seen several “uncertain demand-responsive supply”, but is there any “uncertain supply-responsive demand”? Is this what supply-demand coordinating domain try to do?

Outside transportation, in innovation domain (where abnormal is also normal) has different strategies for supply vs demand-side innovation.

A.
1. incident duration (infrac - stochastic opt (expectation of incidence), some can be captured in discrete modeling vs cont.))
2. alternative routes (availability, absentism)

**Comment by tomfid** [comment]
I'm a little fuzzy on the precise setting here ... 

When I think of a commodity market, like oil, there's definitely randomness and uncertainty on both the supply side and the demand side. On the supply side, it's uncertainty about the supply curve and if you get more granular, randomness of field discovery, development project timing, and interruptions (Ever Given stuck in canal). On the demand side, it's uncertainty about the demand curve, end use tech evolution, with random events (covid, weather) playing a smaller role because the supply side isn't as lumpy (more numerous, diverse and smaller agents).

**Comment by tomfid** [comment]
Given the asymmetry in agent size in a lot of markets, I think upstream sellers are often trying to assess downstream demand, but downstream buyers are not - they're price takers.

**Comment by hyunjimoon** [comment]
questions whose answers i'm aiming to synthesize with @chasfine's input (based on past observation): 
1. phase1 termination diagnostic?
2. was OFE existing need or latent need when yu started?
3. could we decompose scott's entrep.model into feasibility and objective function?
4. projection of om field in 2028?


<img width="1509" alt="image" src="https://github.com/Data4DM/BayesSD/assets/30194633/97a1a597-dacc-483c-916e-9689830087c1">

**Comment by hyunjimoon** [comment]
## what?
can building business simulation tools for startups be automated?
## why important?

[startup need simulation] simulation tool that automatically learns its environment and growth phase to project action to revenue and profit can help align differing opinions on task prioritization, pivot strategy, achievable vision, between entrepreneurs within a startup. This can further align vision, strategy, operation of a startup, realizing simulation to physical reality.
[simulation need automation] business simulation tool help but this differs too much by industry and phase 1,2,3
[automated simulation need startup] business simulation tool is good application tool for simulation-based calibration concept

## now what?

based on identified need and solution pair above, they are matured into paired customer(need) and technology(sol) as below:

## tech(sol)
two approach to modeling internal (startup) and external environments (market, industry) described [here](https://streaklinks.com/Bz3VjZd7-yT8l2_RfQ-v4BbU/https%3A%2F%2Fgithub.com%2FData4DM%2FBayesSD%2Fdiscussions%2F174%23discussioncomment-7922743?email=hyunji.moonb%40gmail.com)

1 simulation-based inference

- tbc with sbc collaborators’ help

- potential connection with Paul's recent paper on [simulation-based prior elicitation](https://arxiv.org/pdf/2308.11672.pdf)?
<img width="863" alt="image" src="https://github.com/Data4DM/BayesSD/assets/30194633/4c3ff74a-1b50-4956-bec6-d91a254a8c08">

- Vikash (probabilistic computing lab at MIT) offered help by showing [this](https://streaklinks.com/Bz3VjaWJCUQnECKUfggNI41Q/https%3A%2F%2Fprobcomp.github.io%2Fgen-finance%2F?email=hyunji.moonb%40gmail.com) business simulation demo of adtech, which I'm generalizing to other industries to learn the structure of general startup operations theory

2 inverse RL
- formulate entrepreneurial decision making as POMDP with:
- hierarchical action a of strategy σ, operations θ, business configuration β
- proxy reward as function of revenue, cost, growth under spatiotemporal demand, productivity, technology uncertainties

## customer(need)
1 as part of singapore smart nation project, I identified government agency customers who showed interest in testing/giving feedback on business simulations tool

2 gained Scott Stern (entrepreneurship academia leader)'s support for distributing my work. When I shared 10pg. elaboration of the above, he highly encouraged me to pursue this and invited me to the bayesian entrepreneurship conference he's hosting this March. This would be a great opportunity to expose simulation + bayesian computation (=sbc) to entrepreneurship domain and further economists which is big market as you may know.

**Comment by hyunjimoon** [comment]
building on #180 
<img width="1358" alt="image" src="https://github.com/Data4DM/BayesSD/assets/30194633/ea776db8-3e6f-42ff-9300-6e2b03f01d50">

**Comment by hyunjimoon** [comment]
Shi Ying, Yichen, Shuqing, Angie

## goal setting
1. targeting Oct.2024 INFORMS as first collaboration milestone

2. how collaboration may be helpful to Angie: verifying practicality of Angie's (sometimes overly theoretical&simplifying) framing existing data in startup world, judging whether it's worthwile to collect certain data  

3. how collaboration may be helpful to Yichen: lack of quantitative data for early (pre-A) stage i.e. inability to do quantitative research with e.g. econometric tools
-> mapping case study texts to action sequence (e.g. n,n,n,n,s,f,f for customer pull) turn qualitative to quantitative via considering simulated multiple paths 

## meeting summary
![image](https://github.com/Data4DM/BayesSD/assets/30194633/f0e1e470-2203-4307-a7cc-40f7c4d422d4)


## action item
4. [short term data] Shi Ying has once classified around 70 harvard case studies to their action sequence, but as the actions were not "uncertainty decreasing" framed, she'll revisit, re-organize, and share 

5. [long term data] Angie will write mail draft to Vishal (VC@global500) who shared interest in developing investment decision making framework based on their data on 1. perspective of candidates, 2. investment choosing heuristic, 3. operational detail on portfolio. This data is more fine grained

Q. perhaps scheduling a meeting sometime in mid Feb. give us enough time to address 4, 5?

<img width="100" alt="image" src="https://github.com/Data4DM/BayesSD/assets/30194633/2bacd8e5-48dd-4f1d-84e1-fa4d8d9688da">

**Comment by hyunjimoon** [comment]
joel mokyr analogize human evolution with technology development. he thinks the concept of "species" from human evolution corresponds with that of "technology", not firm nor societies not the people
[Mokyr92_evoldyn_tech.pdf](https://github.com/Data4DM/BayesSD/files/14207298/Mokyr92_evoldyn_tech.pdf)

quotes:
-  by analogy, technological change will be hampered in an economic environment in which ?rms cannot freely enter or exit an activity, or where they are constrained by traditions and institutions to a fixed share of the market. 
- natural selection works at the level of the technique in use (the analogue of species or deme) as well as on the level of the ?rm (the analogue of individual organism)

some qs:
- q1. from "selection is like a fire that consumes its own fuel…unless variation is renewed periodically, evolution would have come to a stop almost at its inception", -> what is the source of this periodically renewed variation? from unmodeled inward? 
-  technological change ventures into the unknown, not into the uncertain. The risks cannot be diversified away.
